ta/mk/build-user-ta.mk

```diff
@@ -54,3 +54,15 @@ include mk/compile.mk
 # Install TA libraries before in-tree TAs can be linked
 additional-link-deps := $(ta_dev_kit-files-lib)
 include  ta/arch/$(ARCH)/link.mk
+
+.PHONY: install_in_tree_tas
+install_in_tree_tas: $(out-dir)/export-$(ta-target)/bin/$(user-ta-uuid).ta
```

Comment:
Maybe rename to `export_in_tree_tas`.

I think `install_in_tree_tas` depends on `ta_dev_kit`.

Reply:
> Maybe rename to `export_in_tree_tas`.

OK

> I think `install_in_tree_tas` depends on `ta_dev_kit`.

No, it only needs the `.ta` file, nothing else (as you can see the recipe just copies that file). That the `.ta` file depends on many other stuff, which are mostly what is in `ta_dev_kit`, is another story ;-)

Reply:
I thought about the export path being created before TAs can be exported `$(out-dir)/export-$(ta-target)/bin/`.

Also i think TAs should be in  **<export-dir>/ta/** rather than **<export-dir>/bin/**. no?

Reply:
> I thought about the export path being created before TAs can be exported `$(out-dir)/export-$(ta-target)/bin/`

It is taken care of by `mkdir -p`.

> Also i think TAs should be in <**export_dir>/ta/** rather than **<export_dir>/bin/**. no?

I have no strong opinion... Indeed `ta` is less ambiguous than `bin`.

Reply:
> It is taken care of by mkdir -p.

::confused::